docs/content/en/docs/Reference/auroraboot.md

```diff
@@ -0,0 +1,458 @@
+
+## Scope
+
+`AuroraBoot` has the following scope:
+
+- **Download** release assets in order to provision one or more machines
+- **Prepare** automatically the environment to boot from network
+- **Provision** machines from network with a version of Kairos and cloud config
+- **Customize** The installation media for installations from USB
+
+## Prerequisites
+
+- `docker` or a container engine of your choice
+- Port `8090`, `8080` and `67` free on the host running `AuroraBoot`
+- The machine running `AuroraBoot` have to be on the same network segment of the nodes to be bootstrapped
+- The nodes need to be configured to boot over network, or be capable of booting via USB for offline mode
+- `ProxyDHCP` supported by the `DHCP` network attempting to netboot (see also [pixiecore architecture](https://github.com/danderson/netboot/blob/master/pixiecore/README.booting.md#step-1-dhcpproxydhcp)).
+   There should be an already running `DHCP` server in your network. `AuroraBoot` doesn't take over the `DHCP` server, neither require you to do any specific configuration, however a `DHCP` server which is compliant to `ProxyDHCP` requests should be present in the same network running `AuroraBoot` and the machines to boot.
+
+## Overview
+
+To run `AuroraBoot`, simply use `docker` or the container engine of your choice (such as `podman`, ...). `Auroraboot` images are published in [quay](https://quay.io/repository/kairos/auroraboot) and the source code is available [in GitHub](https://github.com/kairos-io/AuroraBoot).
+
+The basic usage of AuroraBoot involves passing it several parameters that define the installation environment, such as the version of Kairos you want to install, the cloud config you want to use, and other customizations you may need. You can pass these parameters either as command-line arguments, or as a full YAML configuration file.
+
+AuroraBoot will download the artifacts required for bootstrapping the nodes, and prepare the environment required for a zero-touch deployment.
+
+For example, to netboot a machine with the latest version of Kairos and Rockylinux using a cloud config, you would run the following command:
+
+```bash
+docker run --rm -ti --net host quay.io/kairos/auroraboot \
+                    --set "artifact_version=v1.5.0" \
+                    --set "release_version=v1.5.0" \
+                    --set "flavor=rockylinux" \
+                    --set repository="kairos-io/kairos" \
+                    --cloud-config https://...
+```
+
+This command will download the necessary artifacts and start the provisioning process. The machine will attempt to boot from network, and will be configured with the specified version of Kairos.
+
+### Network-based bootstrapping
+
+By default `AuroraBoot` will automatically attempt to bootstrap other machines in the same network that are configured for booting from network, with no further configuration or setting necessary.
```

Comment:
I think this makes it easier to read, but please have no hesitation to ignore the suggestion

```suggestion
By default `AuroraBoot` will automatically attempt to bootstrap other machines, which are configured to boot from network, within the same network. No further configuration or settings necessary.
```